examples/with-esm/package.json

```diff
@@ -0,0 +1,21 @@
+{
+  "name": "@envelop-examples/with-esm",
+  "version": "0.0.1",
+  "scripts": {
+    "dev": "tsup-node src/index.ts --format esm --watch src --onSuccess \"node dist/index.mjs\"",
+    "dev:cjs": "tsup-node src/index.ts --format cjs --watch src --onSuccess \"node dist/index.js\"",
+    "start": "tsup-node src/index.ts --format esm --onSuccess \"node dist/index.mjs\"",

```

Comment:
manually adding a `console.log` the index.mjs file of @envelop/core you can see that the ESM support is being used correctly

Reply:
Yeah that was exactly the missing piece for me :) Thanks @PabloSzx